Library/Account.cs

```diff
@@ -44,7 +44,7 @@ public enum AccountState : short
         public bool VatLocationValid { get; private set; }
 
         public Address Address {
-            get { return _address ?? (_address = new Address()); }
+            get { return _address; }
             set { _address = value; }
```

Comment:
Do we need the `_address` anymore? I think we can remove it and change this line to be:

```csharp
public Address Address { get; set; }
```